chatbot.py

```diff
@@ -162,4 +177,13 @@ def message_to_bot(H, clf, learn_response):
             if subj[0] == "geocoding" or subj[0] == location:
                 B = googleMapsApiModule.geocoding(location)
                 learn_response = LearnResponse.MESSAGE.name
+        if (len(subj) != 0 and subj[0] in ["elevation", "height", "depth"]) or (
+            len(adj) != 0 and adj[0] in ["high"]
```

Comment:
'**high**' was treated as an adjective in different types of test sentences and had a POS tag as "**JJ**"